urwid/wimp.py

```diff
@@ -36,7 +36,7 @@
 class SelectableIcon(Text):
     ignore_focus = False
     _selectable = True
-    def __init__(self, text, cursor_position=0):
+    def __init__(self, text, cursor_position=0, *args, **kwargs):
```

Comment:
Better to accept and provide explicit arguments.
In case of long class hierarchy it will be not comfortable to decompose all `*args` `**kwargs` and lookup how unexpected argument reached base class / where need to manually sanitize.

Extra params for docstring can be directly copied from the docstring of parent class